tests/Services/ArrayManagerTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use SwitchViews\Facades\Arr;
+
```

Comment:
Yes, We can do it. But tests should be simple and we should write test cases that are easy to understand. Because we never ever move test cases into production.